types/taiko/index.d.ts

```diff
@@ -440,36 +443,54 @@ export function title(): Promise<string>;
 // https://docs.taiko.dev/api/click
 export function click(
   selector: SearchElement | MouseCoordinates,
-  options?: ClickOptions | RelativeSearchElement,
+  options?: ClickOptions | RelativeSearchElement | HiddenOrDisabledElementsOptions,
```

Comment:
new option has to extend the existing options... For example here user can provide click option along with `HiddenOrDisabledElementOptions`

Reply:
I see, will perform the necessary adjustments.

Reply:
I gave it a go with the more verbose option. At least it compiles. ;)

Reply:
instead can we create a `ForceOption` and extend it in all other options of actions like 

```
export interface ForceOption {
  force: boolean;
}
export interface ClickOptions extends NavigationOptions, ForceOption {}
```

Reply:
Aha! Now I get it, that looks much simpler and cleaner, thanks for the additional hint. Sorry for not seeing this earlier, will likely push a new update this evening (CET).

Reply:
Voilà, implemented as suggested. However, I did not apply your suggested approach to the ```NavigationOptions``` since these are not exclusively used for functions relevant to the new ```ForceOption``` (```clear, focus, hover, mouseAction```) but also for other functions such as ```goto, goBack, goForward```. Therefore, I additionally created this option

```javascript
export interface ForcedNavigationOptions extends NavigationOptions {
  force?: boolean;
}
```

and replaced the ```NavigationOptions``` by it for ```clear, focus, hover, mouseAction```.

Reply:
huh no my suggestion was not to update navigation option..interface can extend multiple interfaces `,` separated
there can be just `ForceOption` which can be extended in interfaces required

Please refer https://github.com/getgauge/taiko/blob/5595c99694f04ecb9b5e4505e207990121f7eaa7/types/taiko/index.d.ts#L47

Reply:
Sorry, what I meant to say was this: if I did what I understood you suggested to NavigationOptions as well, it would look like this, right?
```javascript
export interface NavigationOptions extends BasicNavigationOptions, EventOptions, ForceOption {
  headers?: object;
  waitForStart?: number;
}
```
However, extending the NavigationOptions with ForceOption like above would have the (IMHO negative) side-effect of automatically adding the new ```force: boolean``` option to functions which actually don't support this option (according to the Taiko documentation), such as ```goto(), goBack(), goForward()``` because these functions are also using the NavigationOptions:
```javascript
export function goto(url: string, options?: NavigationOptions): Promise<Response>;
export function goBack(options?: NavigationOptions): Promise<void>;
export function goForward(options?: NavigationOptions): Promise<void>;
```
In order to avoid that, I created
```javascript
export interface ForcedNavigationOptions extends NavigationOptions {
  force?: boolean;
}
```
and added it (by replacing NavigationOptions with ForcedNavigationOptions) to the following functions: ```clear(), focus(), hover(), mouseAction()``` like so:
```javascript
export function focus(selector: SearchElement, options?: ForcedNavigationOptions): Promise<void>;
```
This way, ```clear(), focus(), hover(), mouseAction()``` would now also support the new ```force: boolean``` option (as they should do according to the docs) but ```goto(), goBack(), goForward()``` continue not supporting it (which according to the docs is correct). See what I mean? 😃

Reply:
I see where you are coming from... thanks for the explanation 👍